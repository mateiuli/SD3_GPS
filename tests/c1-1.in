2 1 2.5 5 6 3.5

