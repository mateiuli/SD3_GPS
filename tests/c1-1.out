31.000 1 0 5
